.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,73 @@
+name: Deploy and Configure Morio
+
+on:
+  workflow_dispatch:  # This allows the workflow to be triggered manually
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'  # Replace with the Node.js version you need
+
+      - name: Install AWS CLI
```

Comment:
GitHub Actions runners come with the aws cli pre-installed 😇  no need to install it